### ISSUE TITLE
Node version (nvm)

### ISSUE THREAD

**Issue by MattiaPrimavera** [Instant-Visio/InstantVisio-WebApp#174]
Is there a particular reason to use two different node versions for firebase functions and the front app ?
- functions => `v8.15.0`
- front => `v10.17.0`

**Comment by Abibibi** [comment]
Node 8 is needed for Firebase functions

**Comment by MattiaPrimavera** [comment]
Yep but from [functions docs](https://firebase.google.com/docs/functions/get-started#set-up-node.js-and-the-firebase-cli):

> You'll need a Node.js environment to write functions, and you'll need the Firebase CLI to deploy functions to the Cloud Functions runtime. Node.js versions 8 and 10 are supported. For installing Node.js and npm, Node Version Manager is recommended.

**Comment by HugoGresse** [comment]
Node 10 on GCP is in "beta" but we could indeed switch to 10 on both without issue.

**Comment by MattiaPrimavera** [comment]
Which part of our service is using `GCP` ? Well if it's beta, better not to change that for production ... but the docs I linked seem saying it's supported for `firebase functions`, not beta :)

**Comment by HugoGresse** [comment]
Firebase run on GCP. so Functions are Google Cloud Functions under the shell. 
You can probably make another pr to change the node version on functions and remove the .nvmrc.
